std_execution.bs

```diff
@@ -2563,6 +2611,100 @@ enum class forward_progress_guarantee {
 
     2. Otherwise, `execution::transfer(execution::just(vs...), s)`.
 
+#### `execution::just_error` <b>[execution.senders.just_error]</b> #### {#spec-execution.senders.just_error}
+
+1. `execution::just_error` is used to create a sender that propagates an error to a connected receiver.
+
+    <pre highlight=c++>
+    template&lt;class T>
+    struct <i>just-error-sender</i> // exposition only
+    {
+      T err_;
+
+      template&lt;template&lt;class...> class Tuple, template&lt;class...> class Variant>
+      using value_types = Variant&lt;>;
+
+      template&lt;template&lt;class...> class Variant>
+      using error_types = Variant&lt;T>;
+
+      static const constexpr auto sends_done = false;
+
+      template&lt;class R>
+      struct operation_state {
+        T err_;
+        R r_;
+
+        friend void tag_invoke(execution::start_t, operation_state& s)
+          noexcept {
+          try {
+            execution::set_error(std::move(s.r_), std::move(err_));
+          }
+          catch (...) {
+            execution::set_error(std::move(s.r_), current_exception());
+          }
+        }
+      };
+
+      template&lt;receiver&lt;T> R>
+        requires copyable&ltT>
+      friend auto tag_invoke(execution::connect_t, const <i>just-error-sender</i>& j, R && r) {
+        return operation_state&lt;R>{ j.err_, std::forward&lt;R>(r) };
+      }
+
+      template&lt;receiver&lt;T> R>
+      friend auto tag_invoke(execution::connect_t, <i>just-error-sender</i>&& j, R && r) {
+        return operation_state&lt;R>{ std::move(j.err_), std::forward&lt;R>(r) };
+      }
+    };
+
+    template&lt;<i>moveable-value</i> T>
+      <i>just-error-sender</i>&lt;remove_cvref_t&lt;T> just_error(T && t) noexcept(<i>see-below</i>);
```

Comment:
Do we want to require that the error type is at least `nothow_move_constructible_v<remove_cvref_t<T>>`?

This may be a wider question for what we require/assume with regards to error types sent via `execution::set_error()`.

Reply:
I'd rather we revisit that post-R2.